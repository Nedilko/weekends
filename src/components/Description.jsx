function Description() {
  return (
    <h1 className="font-thin uppercase text-center text-slate-600 dark:text-gray-400 text-5xl mb-10 mt-40 landscape:mt-20">
      weekends countdown
    </h1>
  )
}

export default Description
