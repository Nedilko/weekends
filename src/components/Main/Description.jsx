function Description() {
  return (
    <h1 className="mt-20 text-center text-5xl font-thin uppercase">
      weekends countdown
    </h1>
  )
}

export default Description
