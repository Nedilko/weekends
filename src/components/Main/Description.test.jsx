import Description from '@components/Main/Description'
import { render, screen } from '@utils/test-utils'

describe('text in the document', () => {
  it('title is visible', () => {
    render(<Description />)
    const title = screen.getByRole('heading')
    expect(title).toBeInTheDocument()
    expect(title).toMatchSnapshot()
  })
})
