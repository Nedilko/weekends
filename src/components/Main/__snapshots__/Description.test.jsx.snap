// Vitest Snapshot v1

exports[`text in the document > snapshot 1`] = `
<h1
  class="mt-20 text-center text-5xl font-thin uppercase"
>
  weekends countdown
</h1>
`;

exports[`text in the document > title is visible 1`] = `
<h1
  class="mt-20 text-center text-5xl font-thin uppercase"
>
  weekends countdown
</h1>
`;
